internal/blockchain/ethereum/eventstream.go

```diff
@@ -83,6 +84,7 @@ func (s *streamManager) createEventStream(ctx context.Context, topic string, bat
 		BatchTimeoutMS: batchTimeout,
 		Type:           "websocket",
 		WebSocket:      eventStreamWebsocket{Topic: topic},
+		Timestamps:     true,
```

Comment:
Looks like this is only for new event streams - do we need to handle updating existing streams?

Reply:
Possibly. If right now the only place in the code that uses this is contract events which are net new. So I don't think we need to update any existing event streams for those. If we want to pull blockchain timestamps into existing events I think we do need to update existing event streams. But that seems like it could be a separate piece of work?

Reply:
We reuse a single event stream and just add new subscriptions to it. So the event stream will be pre-existing for any currently running FF stack.

Reply:
Okay pushed some changes that should update existing event streams on startup now. At the moment this only applies to ethconnect. We'll also need to make this change for fabconnect once it supports timestamps as well.